delivery_carrier_service_level/models/stock_picking.py

```diff
@@ -10,14 +10,20 @@ class StockPicking(models.Model):
     _name = "stock.picking"
     _inherit = "stock.picking"
 
+    def _compute_carrier_service_level(self):
+        carrier_model = self.env["carrier.service.level"]
```

Comment:
This is misleading. It is not a carrier model, but a service_level_model, please rename accordingly.